src/modules/simulation/simulator_mavlink/sitl_targets_flightgear.cmake

```diff
@@ -80,5 +86,7 @@ if(ENABLE_LOCKSTEP_SCHEDULER STREQUAL "no")
 				DEPENDS px4 flightgear_bridge
 			)
 		endforeach()
+	else()
+        	message(FATAL_ERROR "Unable to find the executable FlightGear file!")
```

Comment:
```suggestion
```

We can't have this part, it will trigger in general and abort the cmake configure.

You could do it in combination with FG_BINARY being set though.

Reply:
My intent was to abort FlightGear target cmake in case of missing FlightGear binary. 

I do not understand the caused issue by the presence of this FATAL_ERROR message.

Reply:
People should be able to build and use `px4_sitl_nolockstep` without flightgear installed. 

Alternatively we could bake this into a variant like `px4_sitl_flightgear` or something if you want to enforce it at a build time like this.

Does that make sense?